packages/button/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.2.0",
   "description": "Material Components React Button",
   "license": "Apache-2.0",
+  "main": "dist/index.js",
```

Comment:
Probably better as another pr: if you still want an esm build like Dan mentioned, you could run Babel with `["env", { "modules": false" }]` into a new dist type folder and add the `module` field like with main so it just doesn't touch the import/exports.

Reply:
@hzoo I just pushed some changes that doesn't touch the import/exports by adding the {modules: false} options. I think its working correctly as I'm now seeing lines like this:

```js
/* harmony import */ var __WEBPACK_IMPORTED_MODULE_0_react__ = __webpack_require__(0);```